framework/util/CMakeLists.txt

```diff
@@ -177,9 +177,13 @@ endif()
 if (TARGET ZLIB::ZLIB)
     target_compile_definitions(gfxrecon_util
                                PUBLIC
-                                   ENABLE_ZLIB_COMPRESSION
+                                   ENABLE_ZLIB_COMPRESSION ENABLE_PNG_SCREENSHOT
```

Comment:
I admit it would look ugly to have one of these with a project prefix and the other not, but we have talked before about doing a sweep and prefixing build sys vars and macros which are currently not prefixed.

Reply:
Done.